src/thebelab.js

```diff
@@ -164,9 +164,14 @@ function renderCell(element, options) {
   $cell.append($cm_element);
   $cell.append(
     $("<button class='thebelab-button thebelab-run-button'>")
-      .text("run")
+      .text("run this cell")
       .click(execute)
   );
+  $cell.append(
+    $("<button class='thebelab-button thebelab-restart-button'>")
+      .text("restart the kernel")
```

Comment:
Sorry, not `.text`, which changes the button label, but `.attr("title", "restart the kernel")` to give hover text. Leave the button text as "run" and "restart"